model/task_lifecycle.go

```diff
@@ -826,6 +826,9 @@ func getBuildStatus(buildTasks []task.Task) string {
 
 	// started but not finished
 	for _, t := range buildTasks {
+		if t.Status == evergreen.TaskStarted {
+			return evergreen.BuildStarted
+		}
 		if t.Activated && !t.Blocked() && !t.IsFinished() {
```

Comment:
I feel like this line would be clearer to me if it were
```
if evergreen.IsUnstartedTaskStatus(t.Status) && t.Activated && !t.Blocked()
```
It's functionally the same, but it's telling the story that a build is "started" if 
a. any of its tasks is started or
b. if there's at least one finished task and an unstarted task that will eventually run.

Potentially even breaking them into separate for loops could clarify it more so you can see that when you get to the second loop there are no tasks with status started and it's just a mixture of finished and unstarted tasks.

I don't feel strongly about it, but I found it hard to keep in mind a picture of what I'd expect the mixture of task statuses to be at each step.

Reply:
This line actually isn't one that I wrote but a line that you wrote, but personally "activated and not blocked and not finished" does make sense to me, and since it's pre-existing and not broken I'd rather not change it.